### ISSUE TITLE
Claimed: Translate Filter Configuration Example Incorrectly Includes Commas W/ Parameters

### ISSUE THREAD

**Issue by MorrieAtElastic** [logstash-plugins/logstash-filter-translate#63]
Please post all product and debugging questions on our [forum](https://discuss.elastic.co/c/logstash). Your questions will reach our wider community members there, and if we confirm that there is a bug, then we can open a new issue here.

For all general issues, please provide the following details for fast resolution:

- Version: 6.3.0
- Operating System:
- Config File (if you have sensitive info, please remove it):
- Sample Data:
- Steps to Reproduce:

Customer states that example given at https://www.elastic.co/guide/en/logstash/current/plugins-filters-translate.html is wrong and that parameters should not be terminated by commas: 

```
filter {
  translate {
    dictionary => {
      "100"         => "Continue",
      "101"         => "Switching Protocols",
      "merci"       => "thank you",
      "old version" => "new version"
    }
  }
}
```

and that the correct syntax is: 

```
filter {
  translate {
    dictionary => {
      "100"         => "Continue"
      "101"         => "Switching Protocols"
      "merci"       => "thank you"
      "old version" => "new version"
    }
  }
}
```

I am not in position to test this, but if confirmed, the documentation should be changed.

**Comment by guyboertje** [comment]
The claim that the docs are incorrect is correct!
Will fix.

**Comment by guyboertje** [comment]
Fixed with PR #60, not released yet.
